app/src/main/kotlin/org/kotlinlsp/index/Command.kt

```diff
@@ -5,8 +5,9 @@ import org.jetbrains.kotlin.psi.KtFile
 
 sealed class Command {
     data object Stop : Command()
-    data object SourceIndexingFinished: Command()
-    data object IndexingFinished: Command()
+    data object SourceScanningFinished: Command()
+    data object ScanningFinished: Command()
```

Comment:
As we have 2 phases I think these commands should be:
- SourceScanningFinished <- Source files scanned, analysis api can be used
- IndexingFinished <- Background indexing finished (right now this one is here just to have control in junit tests, for example if you want to write a test for autocomplete you can wait for this to finish to have reliable results)